src/Cli/dotnet/commands/dotnet-run/RunCommandParser.cs

```diff
@@ -34,6 +34,11 @@ internal static class RunCommandParser
             Description = LocalizableStrings.CommandOptionNoLaunchProfileDescription
         };
 
+        public static readonly CliOption<bool> NoLaunchProfileArgumentsOption = new("--no-launch-profile-arguments")
```

Comment:
nit: Add `Arity = ArgumentArity.Zero` to the object initializer for this option so that this is treated as a flag-only option.